.ci/test.sh

```diff
@@ -21,15 +21,14 @@ cd $BUILD_DIRECTORY
 if [[ $TASK == "check-docs" ]]; then
     cd $BUILD_DIRECTORY/docs
     conda install -q -y -n $CONDA_ENV -c conda-forge doxygen
-    pip install --user -r requirements.txt linkchecker rstcheck
+    pip install --user -r requirements.txt rstcheck
     # check reStructuredText formatting
     cd $BUILD_DIRECTORY/python-package
     rstcheck --report warning `find . -type f -name "*.rst"` || exit -1
     cd $BUILD_DIRECTORY/docs
     rstcheck --report warning --ignore-directives=autoclass,autofunction,doxygenfile `find . -type f -name "*.rst"` || exit -1
-    # build docs and check them for broken links
+    # build docs
```

Comment:
WDYT? 

```suggestion
    # build docs
    if [[ $TASK == "check-links" ]]; then
        # check docs for broken links
        linkchecker --config=.linkcheckerrc ./_build/html/*.html || exit -1
    fi
```
And above in the code change TASK condition to
```
if [[ $TASK == "check-docs" ]] || [[ $TASK == "check-links" ]]; then
```

Reply:
ok. Changed in https://github.com/microsoft/LightGBM/pull/3890/commits/7374f873f4cd95c489a6e4e54dc38f73b0e62c96. Seems like it worked (https://github.com/microsoft/LightGBM/pull/3890/checks?check_run_id=1815696973)

```text
That's it. 278 links in 300 URLs checked. 0 warnings found. 0 errors found.
```

and that `check-docs` worked as expected: https://github.com/microsoft/LightGBM/pull/3890/checks?check_run_id=1815697125